README.md

```diff
@@ -18,7 +18,21 @@ Customisations of code and i18n files should be made in the copy of `ala-bie` (n
 
 ### Languages
 
-The bie-plugin uses ISO-639 language codes, particularly ISO-639-3, drawn from http://www.sil.org/iso639-3/
+The bie-plugin uses ISO-639 language codes, particularly ISO-639-3, drawn from http://www.sil.org/iso639-3/ and the AIATSIS codes, drawn from https://aiatsis.gov.au/
+
+
+### Common Names Pull
+
+It is possible to "pull" common names with special status into their own section and have them displayed in a special way.
+To do this, use the following configuration settings:
+
+* `vernacularName.pull.active` Set to true for a pull display (*false* by default)
+* `vernacularName.pull.categories` A comma-separated list of status values that will cause the names to be pulled (*empty* by default)
+* `vernacularName.pull.label` The label for the pull section in the names tab (*Special Common Names* by default)
+* `vernacularName.pull.labelDetail` The detail to put into the label title
```

Comment:
default value missing

Reply:
Fixed in https://github.com/AtlasOfLivingAustralia/bie-plugin/pull/170